lib/twilio/call.ts

```diff
@@ -1356,6 +1356,12 @@ class Call extends EventEmitter {
       messageType: messagetype,
       voiceEventSid: voiceeventsid,
     };
+    this._publisher.info('call-message', 'user-defined-message', {
+      call_sid: callsid,
```

Comment:
I think call_sid is auto populated from the call object. Can you try removing it and see if it still gets sent? update onMessageSent as well.